docs/auxil/kwargs_insertion.py

```diff
@@ -45,7 +45,33 @@
     ),
     "",
 ]
-write_timeout_sub = [":attr:`~telegram.request.BaseRequest.DEFAULT_NONE`", "``20``"]
+
+media_write_timeout_deprecation_methods = [
+    "send_photo",
+    "send_audio",
+    "send_document",
+    "send_sticker",
+    "send_video",
+    "send_video_note",
+    "send_animation",
+    "send_voice",
+    "send_media_group",
+    "set_chat_photo",
+    "upload_sticker_file",
+    "add_sticker_to_set",
+    "create_new_sticker_set",
+]
+media_write_timeout_deprecation = [
+    "    write_timeout (:obj:`float` | :obj:`None`, optional): Value to pass to "
+    "        :paramref:`telegram.request.BaseRequest.post.write_timeout`. Defaults to ``20``."
+    "",
+    "",
+    "       .. deprecated:: NEXT.VERSION",
+    "           In future versions, the default value will be changed to "
+    "           :attr:`~telegram.request.BaseRequest.DEFAULT_NONE`.",
```

Comment:
technically the parameter was already changed to be `DEFAULT_NONE`  so we can't say "in future versions"

Reply:
mh, I don't really mean the value behing the "=" in the signature but rather the value that will be passed to `BaseRequest.post`. And that's still 20s after this PR. I tried to specify a bit